lib/read-file.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const fs = require('fs');
+
+module.exports = function(callback){
+  var readFileArray = [];
+  fs.readFile(`${__dirname}/../data/one.txt`, function(err, data){
+    if (err) throw new Error('text one is not being read');
+    readFileArray.push(data.toString('hex', 0, 8));
+    fs.readFile(`${__dirname}/../data/two.txt`, function(err, data){
+      if (err) throw new Error('text two is not being read');
+      readFileArray.push(data.toString('hex', 0, 8));
+      fs.readFile(`${__dirname}/../data/three.txt`, function(err, data){
+        if (err) throw new Error('text three is not being read');
+        readFileArray.push(data.toString('hex', 0, 8));
+        return callback(readFileArray);
```

Comment:
Good job with nesting the callbacks, soon enough we will get out of the ridiculousness that is nested callbacks!